index.d.ts

```diff
@@ -245,6 +247,12 @@ declare module 'snowflake-sdk' {
          * pass the custom credential manager to this option.
          */
         customCredentialManager?: object;
+
+        /**
+         * The option whether the driver loads the proxy information from the environment variable or not
+         * The default value is true. If false, it does not use the proxy.
+         */
+        proxy?: boolean;
```

Comment:
WDYT about the more appropriate name? This is only a flag, and the name is too general. Maybe: useEnvProxy ?

Reply:
The reason why I use 'proxy' is because Axios uses it. I will replace this with an appropriate name.

Reply:
Replaced it with useEnvProxy.